### PR TITLE
replaced Engine.io by Socket.io (my bad)

### PR DESCRIPTION
**Pull request by Cosmitar**
## What does this Pull Request Do?
- replaces Engine.io with Socket.io
## How should this be manually tested?
- npm install (will install socket.io)
- run server
- enter to index
- below session id, screen must show the default response of the server `pong`

### CODE REVIEW

**Comment by loverajoel** [comment]
- [x] Change in `bower.json`

File: views/screen.jade [hunk 1]
```diff
@@ -1,7 +1,7 @@
 extends layout
 
 block scripts
-  script( src='/js/libs/engine.io/engine.io.js' )
+  script( src='/socket.io/socket.io.js' )
```
Comment (loverajoel) [inline views/screen.jade:4]: why you change the path?

Comment (Cosmitar) [inline views/screen.jade:4]: because socket.io has its own javascript client. The same as on backend, we must replace engine-client by socket-client

**Comment by loverajoel** [comment]
:shipit:

**Pull request merged**